### PR TITLE
[minor_change] Add support for name attribute in mso_schema_template_external_epg_subnet module (DCNE-59)

### LINKED ISSUE #512: Add support for Name in module mso_schema_template_external_epg_subnet
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

* xxxx

### New or Affected Module(s):

* mso_XXXXX

### MSO version and MSO Platform

* V x.x.x and Docker Swarm OVA-based/SE-based/ND-based/all.

### APIC version and APIC Platform for Site Level Resources

* V x.x.x and on-prem/cloud-aws/cloud-azure/all.

### Collection versions

* cisco.mso x.x.x

### References

* #0000

### PR DESCRIPTION
**Pull request by akinross**
fixes #512

### CODE REVIEW

File: tests/integration/targets/mso_schema_template_external_epg_subnet/tasks/main.yml [hunk 1]
```diff
@@ -343,6 +346,14 @@
     - add_epg_subnet_no_ag.msg == "MSO Error 400{{':'}} ExternalEPG{{':'}} ansible_test_1 in Schema{{':'}} ansible_test , Template{{':'}} Template1 External EPG validation error{{':'}} aggregate should be enabled only if shared-rtctrl is enabled in Scope for subnet 10.0.0.2/24"
```
Comment (lhercot) [inline tests/integration/targets/mso_schema_template_external_epg_subnet/tasks/main.yml:346]: Update task title to reflec the additional <4.3 limit

**Review by lhercot — CHANGES_REQUESTED** [review]

**Review by lhercot — APPROVED** [review]
LGTM

**Review by shrsr — APPROVED** [review]
LGTM

**Review by gmicol — APPROVED** [review]
LGTM

**Review by samiib — APPROVED** [review]
LGTM

**Pull request merged**